doc/Overview/Localization/LocalizationOverview.md

```diff
@@ -1,46 +1,51 @@
 ---
 uid: Overview.Localization
 ---
+
 # Localization
-Uno.Extensions.Localization uses [Microsoft.Extensions.Localization](https://www.nuget.org/packages/Microsoft.Extensions.Localization) for any localization related work.
 
-For more documentation on localization, read the references listed at the bottom.
+It is often necessary to adapt an application to a specific subset of users within a market. **Localization** includes the type of actions developers take to modify both user interface elements and content to adhere to more languages or cultures. Specifically, text **translation** is done by applying alternate strings of text at runtime which accomodate a user's language preference. Many apps store these pieces of text in dedicated resource files that the app parses and assigns as text throughout the application. `Uno.Extensions.Localization` provides a consistent way to resolve text of a specific culture or locale across platforms. This feature allows for modifying them to be applied upon app restart.
 
-## Text localization
+It uses [Microsoft.Extensions.Localization](https://www.nuget.org/packages/Microsoft.Extensions.Localization) for any localization related work. For documentation on the broader process of localization, read the references listed at the bottom.
```

Comment:
After this initial overview, let's start with a short section showing
- How to enable localization eg UseLocalization
- How to add different language resources